physlr/physlr.py

```diff
@@ -997,15 +1032,17 @@ def physlr_mst(self):
     def physlr_backbone(self):
         "Determine the backbone path of the graph."
         g = self.read_graph(self.args.FILES)
-        backbones = self.determine_backbones(g)
+        backbones = self.determine_backbones(g, self.args.junction_threshold)
         for backbone in backbones:
             print(*backbone)
 
     def physlr_backbone_graph(self):
         "Determine the backbone-induced subgraph."
         g = self.read_graph(self.args.FILES)
         Physlr.remove_singletons(g)
-        backbones = Physlr.determine_backbones_and_remove_chimera(g)
+        print(int(timeit.default_timer() - t0),
+              "Determining the backbone-induced subgraph.", file=sys.stderr)
```

Comment:
Either remove this `print` or move it to the function `determine_backbones`.

Reply:
I moved it to `determine_backbones_and_remove_chimera` as `determine_backbones` might be called more than once.